### PR TITLE
Added new endpoint actions

### PR DESCRIPTION
**Pull request by RamboV**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [x] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Security Considerations (REQUIRED)
- [x] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers), 
  please document them in the `readme.html`.
- [x] If this is a new connector or you are adding new actions
    - [x] Please document in the `readme.html` all methods (eg, OAuth) used to authenticate 
      with the service that the connector is integrating with.
    - [ ] If any actions are unable to run on SOAR Cloud, please document this in the `readme.html`.
- [ ] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [ ] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through 
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [ ] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

  

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

File: release_notes/unreleased.md [hunk 1]
```diff
@@ -1 +1,2 @@
 **Unreleased**
+* Added new actions: lookup dynamicdns domain, lookup malware record ip, lookup malware record domain, lookup ssl certificate domain, lookup os indicator domain, lookup os indicator ip [PAPP-28033]
```
Comment (dgopani-crest) [inline release_notes/unreleased.md]: Added new actions `lookup dynamicdns domain`, `lookup malware record ip`, `lookup malware record domain`, `lookup ssl certificate domain`, `lookup os indicator domain`, and `lookup os indicator ip`

**Review by dgopani-crest — APPROVED** [review]

**Review by dhwanis-crest — APPROVED** [review]

File: hyasinsight.json [hunk 2]
```diff
@@ -11,11 +11,11 @@
     "product_version_regex": ".*",
     "publisher": "Hyas",
     "license": "Copyright (c) Hyas, 2022",
-    "app_version": "1.1.0",
+    "app_version": "1.2.0",
     "utctime_updated": "2022-06-29T08:24:29.290348Z",
     "package_name": "phantom_hyasinsight",
     "main_module": "hyasinsight_connector.py",
-    "min_phantom_version": "5.2.0",
+    "min_phantom_version": "5.3.0",
```
Comment (mmomin-crest) [inline hyasinsight.json]: Update min_phantom_version value as 5.3.4.

**Review by mmomin-crest — APPROVED** [review]

**Review by splunk-jessica — APPROVED** [review]

**Comment by RamboV** [comment]
@splunk-soar-connectors-bot @dgopani-crest @splunk-jessica 
The App is shown as not supported instead of "Developer supported " in splunk base, can you guide us what needs to be done

**Comment by nastorSplunk** [comment]
You need to join our partner program to get control of the Splunkbase pages and GitHub repos for the apps you have created. We have a [blog post](https://www.splunk.com/en_us/blog/security/publish-your-splunk-soar-apps-faster.html) and a [guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/PARTNERS.md), fill out the form at the bottom of the blog post to get started.

**Pull request merged**